setup.py

```diff
@@ -66,7 +66,7 @@
         package_dir={
             '': 'lib',
         },
-        scripts=['bin/pint'],
+        scripts=['bin/pint', 'bin/awscsvgen'],
```

Comment:
If this is added back to setup then the boto requirement should be included in requirements file. But this is going to cause problems in build service for suse_version <= 1320 due to requirements mismatch with spec.

Reply:
I.e. the package would be built without requires on boto. But the egg would have the requirement so pint/awscsvgen would fail invocation.